src/Microsoft.Identity.Client/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -112,7 +112,7 @@ private string GetTeamId()
 
         public iOSTokenCacheAccessor()
         {
-            keychainGroup = GetTeamId() + '.' + GetBundleId();
+            keychainGroup = GetTeamId() + '.' + DefaultKeychainGroup;
```

Comment:
Is it related to this PR?

Reply:
no, it was test code and should be removed.